packages/schemas/src/player/gamemodes/general/quests/modes/cops-and-crims.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { APIData, removeFormatting } from "@statsify/util";
+import { Field } from "../../../../../metadata";
+import { FormattedGame } from "../../../../../game";
+import { GameQuests } from "../game-quests";
+import { QuestTime, getAmountDuring, questFieldData } from "../util";
+import { add } from "@statsify/math";
+
+export class CopsAndCrimsQuests implements GameQuests {
+  @Field({ ...questFieldData, leaderboard: { name: "Win a game! (Defusal)" } })
+  public winAGame: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "Kill 15 players! (Defusal)" } })
```

Comment:
The leaderboard metadata won't get spread properly here because you overwrite the leaderboard field.